app/components/HeaderNotifications/HeaderNotifications.css

```diff
@@ -22,10 +22,13 @@ html[data-theme='dark'] .notifications {
 
 .notification {
   border-bottom: 1px solid lightgray;
-  padding: 15px;
+  padding-top: 15px;
+  padding-right: 15px;
+  padding-left: 15px;
```

Comment:
This is the same as `padding: 15px 15px 0`

Reply:
fixed